src/frontend/utils/kalender/io.ts

```diff
@@ -0,0 +1,70 @@
+import dayjs from 'dayjs';
+
+import { datoformat } from '../formatter';
+import { antallDagerIMåned, DagMånedÅr, FamilieIsoDate } from './typer';
+import { capString } from './utils';
+
+export const parseIso8601String = (dato: FamilieIsoDate): DagMånedÅr => {
+    const dayjsDato = dayjs(dato, datoformat.ISO_DAG);
+
+    // Dayjs brukes ikke til noe annet enn å fange ugyldige inputs
+    if (!dayjsDato.isValid()) {
+        throw new Error(`Dato '${dato}' er ugyldig`);
+    }
+
+    const år: number = parseInt(dato.substr(0, 4), 10);
+    const måned: number = parseInt(dato.substr(5, 7), 10);
+    const dag: number = parseInt(dato.substr(8, 10), 10);
+
+    if (år < 1800 || år > 2500) {
+        throw new Error(`År fra dato '${dato}' er '${år}' og er sannsynligvis feil`);
+    }
```

Comment:
Tror du har dekning for den påstanden ja. Greit å ta i litt 😄 
Er det forresten noen sjanse for at TIDENES_MORGEN/-ENDE blir sendt inn her og bør håndteres spesielt?

Reply:
Hmm, ja kanskje man skulle brukt årene derfra bare? Selv om de også er tullete høye/lave da? Tror egentlig det er greit om det smeller om noen sender inn slike datoer, det gir jo ingen mening 😆